sdk/php/src/Client/AbstractScalar.php

```diff
@@ -4,12 +4,11 @@
 
 namespace Dagger\Client;
 
-use Stringable;
-
-abstract readonly class AbstractScalar implements Stringable
+abstract readonly class AbstractScalar implements \Stringable
 {
-    public function __construct(private string $value)
-    {
+    final public function __construct(
```

Comment:
In this instance, `final` prevents any inheriting class from overriding the method.

This is desirable, if AbstractScalars could override this method we would not be able to depend on it to reconstruct them later.